konoha/word_tokenizers/mecab_tokenizer.py

```diff
@@ -84,12 +121,16 @@ def __init__(
         if dictionary_format is None:
             if system_dictionary_path is None or "ipadic" in system_dictionary_path.lower():
                 self._parse_feature = parse_feature_for_ipadic
+            elif system_dictionary_path is None or "unidic" in system_dictionary_path.lower():
```

Comment:
```suggestion
            elif "unidic" in system_dictionary_path.lower():
```

If `system_dictionary_path` is `None`, konoha assumes users attempt to create a tokenizer with `mecab-ipadic`. Therefore, it is not needed to check if `system_dictionary_path` is empty for `unidic`.